src/CmsAdmin/Form/CategoryAclForm.php

```diff
@@ -22,9 +22,11 @@
 class CategoryAclForm extends \Cms\Form\Form
 {
 
+    public const SCOPE_CONFIG_OPTION_NAME = 'scope';
+
     public function init()
     {
-        $tree = (new \Cms\Model\CategoryModel(new CmsCategoryQuery()))->getCategoryTree();
+        $tree = (new \Cms\Model\CategoryModel((new CmsCategoryQuery())->whereTemplate()->like($this->getOption(self::SCOPE_CONFIG_OPTION_NAME) . '%')))->getCategoryTree();
```

Comment:
to naprawdę musi się wszystko dziać w jednej linijce? ;)

Reply:
oszczednosc cykli procesora, kompilator sie mniej napracuje :D

Reply:
ogarnie się